src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -53,6 +54,37 @@ g.test('duplicate_bindings')
     }, !_valid);
   });
 
+const kMaxBindingNumber = kLimitInfo.maxComputeWorkgroupsPerDimension.default;
```

Comment:
`maxComputeWorkgroupsPerDimension` is not the correct limit here. There is currently no corresponding limit for max binding number... but we actually just agreed to make this a configurable limit, and lower its default value to 640 due to driver issues seen by some implementations. Spec change is here: https://github.com/gpuweb/gpuweb/pull/3318

Let's go ahead and change this test to use 640 now. Eventually, we will want to add a test case where we ask for a higher `maxComputeWorkgroupsPerDimension`, but we don't have a way to do that right now (because we need to know whether the adapter supports a higher limit). We can add a TODO for it.

Reply:
Ah, I see. Then, let me change this test to use 640. But, please correct me if 640 should be allowed.

> Let's go ahead and change this test to use 640 now. Eventually, we will want to add a test case where we ask for a  higher maxComputeWorkgroupsPerDimension, but we don't have a way to do that right now (because we need to  know whether the adapter supports a higher limit). We can add a TODO for it.

ok, I simply add a TODO for it like this "TODO: Check if a higher maximum binding limit can be allowed."